R/json_nest_join.R

```diff
@@ -0,0 +1,29 @@
+#' JSON nest join
+#'
+#' @description
+#' `r lifecycle::badge("experimental")`
+#'
+#' A wrapper around `dplyr::nest_join()` which stores the joined data into a JSON column.
```

Comment:
Here and elsewhere?

```suggestion
#' A wrapper around [dplyr::nest_join()] which stores the joined data into a JSON column.
```

Reply:
done in https://github.com/cynkra/dm/pull/918/commits/80af53f925d7fbd3359d353dc957a4afb17487a1